config/kn-event.yaml

```diff
@@ -0,0 +1,50 @@
+---
+config:
+  branches:
+    release-1.11:
+      openShiftVersions:
+        - skipCron: true
+          version: '4.15'
+    release-1.13:
+      openShiftVersions:
+        - version: '4.15'
+    release-1.14:
+      openShiftVersions:
+        - version: '4.15'
+    release-next:
+      openShiftVersions:
+        - version: '4.15'
+repositories:
+  - org: openshift-knative
+    repo: kn-plugin-event
+    imagePrefix: client-plugin-event
+    binaryBuildCommands: |-
+      export HOME=/tmp
+      export TEST_IMAGES_DIR=/usr/bin
+      ./mage build
```

Comment:
In case of client, I've been looking at this stanza and my thought's been to wrap it into Dockerfile stage build to simplify CI config. Similarly to `Makefile` PR in the kn-plugin-event midstream, but also execute the make withing Docker builds.

Reply:
This is how Openshift CI advises building binaries in a separate step, and later use the `bin` image contents, while building the images.

Since then, most Serverless repos moved to pure Docker approach, when the `bin` building doesn't matter, as everything is done in the build image step: https://github.com/openshift-knative/serving/blob/release-next/openshift/ci-operator/knative-images/activator/Dockerfile

IDK, if I keep kn-event current setup or shift it to how other repos does this. I'm leaning towards the pure Dockerfile approach.

Reply:
I changed my mind to keep the current approach, as I managed to change the code to support both ways. Also, the current kn-event's approach may be better for multi-arch builds in Openshift CI.

Still, maybe, I'll follow up the kn-event to add a Makefile target that detects Openshift CI environment and sets proper environment variables.